Feature/8 Hero Section

KyneLaggui opened the pull request:
## ✨ What’s New?
- Added the Arduino Day 2025 Texts, Background, and Logo
- Added the ability to follow the mouse cursor

## 📷 Screenshots (IMPORTANT)

## Desktop View
![image](https://github.com/user-attachments/assets/07160a5d-3662-44c2-b347-ad984a099d6a)

## Mobile View
![image](https://github.com/user-attachments/assets/0dbf5fe6-1b32-441f-937a-417b8651a222)


## 🔗 Related Issues

Closes #8 


## ✅ Checklist

- [ ] Code follows project conventions.
- [ ] Unused code, variables, and console logs have been removed.
- [ ] Code compiles and runs without errors.
- [ ] Build and tests succeeds locally.
- [ ] Assigned at least one reviewer.

KyneLaggui committed:
feat: added the logo of arduino ph

KyneLaggui committed:
feat: added the cover for the logo of hero section

KyneLaggui committed:
feat: added the ability to make the logo tilt based on the mouse cursor

KyneLaggui committed:
fix: move the logo and hero section cover to public assets

KyneLaggui committed:
feat: added the text for header

KyneLaggui committed:
style: change the pixels to default tailwind styling

KyneLaggui committed:
fix: revert the text sizes to pixels

KyneLaggui committed:
style: adjusted the sizes for responsiveness

KyneLaggui committed:
feat: added moveX and moveY variables on the logo

KyneLaggui committed:
Merge branch 'dev' of https://github.com/ArduinoDayPH2025/arduino-ph-2025 into feature/8-hero-section

szy-kenn committed:
feat(hero): add Hero component in the root page.tsx

szy-kenn reviewed (changes requested):
**Interactive logo works well!**

One issue with the image, hindi lumalabas yung rounded corners and hindi same yung napapakita in mobile view

No rounded corners:
![image](https://github.com/user-attachments/assets/04878eda-31b6-439e-a53e-ef48c9b850d0)

Mobile view differences:

Website:
![image](https://github.com/user-attachments/assets/11254d4c-4cf2-46fd-aba3-2397614308f6)

Figma:
![image](https://github.com/user-attachments/assets/00eab13e-929b-4ed0-b0b0-8ac04fc7e966)

I tried some adjustments din and it works when you remove `objectFit="cover"` sa `<Image />` but it gets squeezed/stretched naman kapag nagbabago yung screen width.

(it shows the rounded corners)
![image](https://github.com/user-attachments/assets/d3e9c05e-48a7-4b5f-9008-736029dcc98b)

(same look sa Figma, but the image is pixelated and squeezed)
![image](https://github.com/user-attachments/assets/9ec6e6a7-02a8-43f0-95d0-ff87869a46fa)

If possible, maybe we can implement the background in code, or if may pwede pang magawang adjustments to fix it @KyneLaggui thank you!

KyneLaggui committed:
feat: added hero section mobile cover picture

KyneLaggui committed:
feat: added the ability to change image based on size

KyneLaggui committed:
feat: added gradient as background

KyneLaggui committed:
Merge branch 'feature/8-hero-section' of https://github.com/ArduinoDayPH2025/arduino-ph-2025 into feature/8-hero-section

KyneLaggui commented:
@szy-kenn 
## Fixed 
- Removed `objectFit="cover"` on the `<Image />` to make the border radius same on the figma file
- Added two images depending on screen sizes 

## Feature
- Added gradient as background 

## Images
### Desktop View
![image](https://github.com/user-attachments/assets/1f6dfbec-38a0-4bf6-954a-7551e9649f19)
### Mobile View
![image](https://github.com/user-attachments/assets/746bac95-5132-4fa3-a95b-17893fd0b453)

xialuna commented on src/components/features/Hero.tsx:
Don't use container class on this, mag custom padding ka nalang (refer on figma)

<img width="755" alt="image" src="https://github.com/user-attachments/assets/87e0d41b-0ea0-4348-bcff-62762eb62c1a" />

xialuna committed:
Merge branch 'dev' into feature/8-hero-section
